mozilla_repo_urls/parser.py

```diff
@@ -21,7 +21,22 @@ def parse(url_string):
     if not parsed_url.valid:
         raise InvalidRepoUrlError(url_string)
 
-    if parsed_url.platform not in _SUPPORTED_PLAFORMS:
-        raise UnsupportedPlatformError(url_string, platform, _SUPPORTED_PLAFORMS)
-
+    if parsed_url.platform not in _SUPPORTED_PLATFORMS:
+        """
+        For error reporting purposes, the exception object includes the domain
+        for each supported platform.
+        """
+        raise UnsupportedPlatformError(
+            url_string,
+            parsed_url.host,
+            [
+                domain
+                for domains in [
+                    platform[1].DOMAINS
+                    for platform in giturlparse.platforms.PLATFORMS
+                    if platform[0] in _SUPPORTED_PLATFORMS
+                ]
+                for domain in domains
+            ],
```

Comment:
Refactor: I love list comprehensions and I don't mind having nested ones. That said, I had some trouble to figure out what the final result looks like. I think we can extract that as a named module constant, since all values are already defined at the module level. What do you think? 

```suggestion
            _SUPPORTED_HOSTS,
```